### ISSUE TITLE
Panic trying out new Python pip support

### ISSUE THREAD

**Issue by kujenga** [fossas/fossa-cli#124]
I got the following panic when trying out the new Python functionality. I tried paring the `.fossa.yml` file in the repo I was experimenting in just to a single, simple requirements.txt file, and was still unable to resolve the panic.

```
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/fossas/fossa-cli/builders.(*PipBuilder).Analyze(0xc4201ad540, 0xc4200233b0, 0xb, 0x174c486, 0x3, 0xc420026910, 0x48, 0xc4200268c0, 0x37, 0x0, ...)
	/Users/kevin/go/src/github.com/fossas/fossa-cli/builders/pip.go:104 +0x93e
main.doAnalyze(0xc42011f600, 0xe, 0xe, 0xc420023300, 0x6, 0xc420020cc0, 0x21)
	/Users/kevin/go/src/github.com/fossas/fossa-cli/cmd/fossa/analyze.go:83 +0x37d
main.analyzeCmd(0xc42011adc0)
	/Users/kevin/go/src/github.com/fossas/fossa-cli/cmd/fossa/analyze.go:25 +0x109
github.com/fossas/fossa-cli/vendor/github.com/urfave/cli.HandleAction(0x162c600, 0x177b100, 0xc42011adc0, 0xc420090800, 0x0)
	/Users/kevin/go/src/github.com/fossas/fossa-cli/vendor/github.com/urfave/cli/app.go:492 +0x7c
github.com/fossas/fossa-cli/vendor/github.com/urfave/cli.Command.Run(0x174f454, 0x7, 0x0, 0x0, 0x0, 0x0, 0x0, 0x175dc2b, 0x1a, 0x0, ...)
	/Users/kevin/go/src/github.com/fossas/fossa-cli/vendor/github.com/urfave/cli/command.go:210 +0xa36
github.com/fossas/fossa-cli/vendor/github.com/urfave/cli.(*App).Run(0xc4200b1d40, 0xc42000c080, 0x2, 0x2, 0x0, 0x0)
	/Users/kevin/go/src/github.com/fossas/fossa-cli/vendor/github.com/urfave/cli/app.go:255 +0x6a0
main.main()
	/Users/kevin/go/src/github.com/fossas/fossa-cli/cmd/fossa/main.go:152 +0x2481
```

**Comment by xizhao** [comment]
I believe this may have something to do with our constraints on parsing only `pip freeze` requirements output `==`.

We should be able to just send unresolved revision specs back to FOSSA, i.e. include the `==3.2` or `>=4.5` constraints

**Comment by ilikebits** [comment]
This should be fixed in latest (v0.7.0). Please let me know if it doesn't work for you.

**Comment by liftM** [comment]
Closing due to inactivity.
